cmd/winch/main.go

```diff
@@ -138,10 +136,8 @@ func main() {
 		tlsConfig,
 		logEntry,
 		mux,
+		*flagDebugMode,
 	)
-	if *flagDebugMode {
```

Comment:
why is this change here

Reply:
to enable debug mode consistently (same for winch and grpc)